geomesa-native-api/src/main/scala/org/locationtech/geomesa/api/AccumuloGeoMesaIndex.scala

```diff
@@ -196,9 +196,9 @@ object AccumuloGeoMesaIndex {
   private def buildSimpleFeatureType[T](name: String)
                                        (view: SimpleFeatureView[T] = new DefaultSimpleFeatureView[T]()) = {
     val builder = new SftBuilder()
-      .date("dtg", true, true)
-      .bytes("payload", new SftBuilder.Opts(false, false, false, Cardinality.UNKNOWN))
-      .geometry("geom", true)
+      .date("dtg", index = false, default = true)
```

Comment:
I'm conflicted on this one.

@anthonyccri what say you?  Should index dtgs by default?